docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/configure-algolia.md

```diff
@@ -36,11 +36,19 @@ To configure Algolia, do the following:
 
 ![algolia-settings](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-settings.png)
 
+11. If your store model includes products without prices, you need to select **Products without prices**.
```

Comment:
Provide a screenshot showing the same as on the screenshot above. Stage doesn't have this setting yet, so can't make one myself

Reply:
![Screenshot from 2025-03-20 11-21-42](https://github.com/user-attachments/assets/6df9cd24-618e-4786-8767-c5d79b2e77cf)
![Screenshot from 2025-03-20 11-21-48](https://github.com/user-attachments/assets/a0350339-8d60-4559-9603-b13ba28ff4e8)

Reply:
![Screenshot from 2025-03-20 11-23-14](https://github.com/user-attachments/assets/3a035cea-e3e0-43ea-a3f8-6edb00aae861)
![Screenshot from 2025-03-20 11-23-20](https://github.com/user-attachments/assets/a1de4685-af17-4992-8239-ec810c24c8ff)